tools/firtool/firtool.cpp

```diff
@@ -472,7 +472,7 @@ static bool checkBytecodeOutputToConsole(raw_ostream &os) {
 /// requested and politely avoiding dumping to terminal unless forced.
 static void printOp(Operation *op, raw_ostream &os) {
   if (emitBytecode && (force || !checkBytecodeOutputToConsole(os)))
-    writeBytecodeToFile(op, os, getCirctVersion());
+    writeBytecodeToFile(op, os, BytecodeWriterConfig(getCirctVersion()));
```

Comment:
hmm, I didn't seem to need to do this.

Reply:
I definitely needed this, and I just double checked that it won't compile without this change, but it's something that I only needed to do after picking up a very recent commit (within the last 24 hours): https://github.com/llvm/llvm-project/commit/6ab2bcffe45e660a68493e6a7cd04b6f05da51dc